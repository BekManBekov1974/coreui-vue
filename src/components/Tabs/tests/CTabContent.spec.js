import { mount } from '@vue/test-utils'
import Component from '../CTabContent'

const ComponentName = 'CTabContent'
const defaultWrapper = mount(Component)


describe(ComponentName, () => {
  it('has a name', () => {
    expect(Component.name).toMatch(ComponentName)
  })
  it('renders correctly', () => {
    expect(defaultWrapper.element).toMatchSnapshot()
  })
})
