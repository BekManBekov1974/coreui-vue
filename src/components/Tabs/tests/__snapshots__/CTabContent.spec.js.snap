// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CTabContent renders correctly 1`] = `
<div>
  
</div>
`;
